media/css/legal/impressum.less

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+#tabzilla:before {
+    background-color: @textColorSecondary;
+}
+
+#main-feature {
+    > * {
+        .span(8);
+    }
+    padding-top: @baseLine;
+    padding-bottom: @baseLine * 2;
+}
+
+#main-content {
+    .intro,
+    .archive {
+        .span(8);
+    }
+    .contents {
+        .span-all();
+    }
+    .main-license {
+        .trailing-arrow();
+        .open-sans-light();
+        .font-size(18px);
+    }
+    .intro {
+        margin-bottom: @baseLine * 2;
+    }
+}
+
+/* Tablet & Mobile */
+
+@media only screen and (max-width: @breakDesktop) {
+
+    #main-feature > *,
+    #main-content .intro,
+    #main-content .archive {
+        .span-all();
+    }
+
+}
```

Comment:
Testing this in dev tools, it seems all we need for the CSS is the following:

```
#tabzilla:before {
    background-color: @textColorSecondary;
}
#main-content {
    h1 {
        padding: 20px 0 40px;
    }
    .intro {
        .span(8);
    }
}

@media only screen and (max-width: @breakDesktop) {
    #main-content {
        .intro {
            .span-all();
        }
    }
}
```
This of course assumes that ``#main-feature` is also removed from the HTML